x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/components/curation_queries/utils.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const filterEmptyQueries = (queries: string[]) => {
```

Comment:
This is totally fine as a helper ... just a note that this is such a small function I was surprised to see it as a utility.

Reply:
Just IMO, doing so:

- helps highlight that we're manually transforming data (at least pre-submit) and not leaving it as-is with empty strings.
- makes it easier to unit test separately
- makes it easier to highlight when we use these types of small utilities & potentially combine them/DRY them out if we start using them in more than one place